os/kernel/log_dump/log_dump.c

```diff
@@ -180,13 +188,23 @@ int log_dump_read_wake(void)
 	int msg_compress = true;
 	ret = mq_send(mq_fd, (const char *)&msg_compress, sizeof(int), 100);
 	if (ret < 0) {
-		lldbg("failed mq_send ret = %d\n", ret);
+		ldpdbg("failed mq_send ret = %d\n", ret);
 		return ERROR;
 	}
 
 	/* wait for the completion of the partially filled block compression */
+#ifdef CONFIG_LOG_DUMP_DEBUG_DETECT_HANG
+	int hang_check_count = 0;
+#endif
 	while (compress_last_block) {
-		usleep(10000);
+		usleep(LOG_DUMP_COMP_CHECK_RETRY_USEC);
```

Comment:
This should be in the conditional. Because the definition is defined in the conditional. I think you dont build test without the config enabled.

Reply:
oh right, I fixed and test without config enable.